ios/customer_io.podspec

```diff
@@ -15,7 +15,8 @@ A plugin for Customer.io
   s.source           = { :path => '.' }
   s.source_files = 'Classes/**/*'
   s.dependency 'Flutter'
-  s.platform = :ios, '9.0'
+  s.dependency 'CustomerIO/Tracking'
```

Comment:
Is this file used for the example Flutter iOS app? Or is this the file used for us to push the flutter SDK to cocoapods so our flutter customers can use it in their app? 

If it's the cocoapods file that we use for publishing, I would like to make some changes to the file to match the naming style of our iOS SDK so the SDK is easier to find.

Reply:
This file is used by iOS internally only, it fetches CustomerIO native SDK for us that we make native bindings with, but i'll dig down more once i start looking into publishing the flutter plugins.